pkg/config/load.go

```diff
@@ -37,6 +42,27 @@ type Prowgen struct {
 	// MultiArchBranchFilter is a filter of branches that will be built for multiple architectures.
 	// If empty, all branches will be included.
 	MultiArchBranchFilter []string `json:"multi_arch_branch_filter,omitempty"`
+	// SlackReporterConfigs defines all desired slack reporter info for included jobs
+	SlackReporterConfigs []SlackReporterConfig `json:"slack_reporter,omitempty"`
+}
+
+// SlackReporterConfig groups test names to a channel to report; mimicking Prow's version, with some unnecessary fields removed
+type SlackReporterConfig struct {
+	Channel           string                `json:"channel,omitempty"`
+	JobStatesToReport []prowv1.ProwJobState `json:"job_states_to_report,omitempty"`
+	ReportTemplate    string                `json:"report_template,omitempty"`
+	JobNames          []string              `json:"job_names,omitempty"`
+	//TODO(sgoeddel): if desired, we could add a list of excluded or included variants here to limit which jobs
```

Comment:
Filtering the variants makes sense. Let's include this change in this PR as well.

Reply:
Good to hear it is reasonable. I would like to do that as a follow up though, if that is okay with you.

Reply:
Follow-up SGTM